### ISSUE TITLE
LV2 version (for Linux users)

### ISSUE THREAD

**Issue by ericfont** [osoumen/C700#7]
I'd be willing to port this from VST format into LV2 so that Linux users can use it.

**Comment by osoumen** [comment]
@ericfont I may not be able to help you, but if it works out, may it be incorporated into our master?

**Comment by ericfont** [comment]
Yes of course I will make PR here if I can get it working.  Hardest part is probably going to be the GUI, which I'll probably have to rewrite into a cross-platform toolkit like Qt.

**Comment by osoumen** [comment]
@ericfont OK. You can freely modify it based on what you fork from our repository. Good luck.
